### ISSUE TITLE
Adding new category for jekyll framework plugins?

### ISSUE THREAD

**Issue by skatkov** [rubytoolbox/rubytoolbox#608]
I'm suggesting considering adding a new category for static-website generator plugin. 

There are at least 3 different static website frameworks -- middleman, nanoc and jekyll. And all of those have eco-system of plugins around them.

https://rubygems.org/search?utf8=%E2%9C%93&query=nanoc
https://rubygems.org/search?utf8=%E2%9C%93&query=jekyll
https://rubygems.org/search?utf8=%E2%9C%93&query=middleman

It seems reasonable to have separate categories for them.

**Comment by colszowka** [comment]
@skatkov Absolutely, feel free to add the corresponding categories as PRs to the catalog. I would suggest to keep it under the group that currently static website generators live under, "Content Management & Blogging"

**Comment by ChaelCodes** [comment]
Do y'all still want this?

I'm starting with the Jekyll one, and there's roughly 2,000.

I filtered to ones updated since 2022, and with at least 8,000 downloads, and there's still 788.

Is this going to be reviewable or mergeable?

**Comment by ChaelCodes** [comment]
I made it to page 5, and then realized there's 27 pages. 😭
